src/modules/lyrics/providers.js

```diff
@@ -277,6 +290,21 @@ BetterLyrics.LyricProviders = {
     };
   },
 
+  formatPlainLyrics: function (plainLyrics) {
```

Comment:
Could you also use this function for the [`ytLyrics`](https://github.com/boidushya/better-lyrics/pull/136/files#diff-9703dcf6fcaf4f8d9037e51121e1a9a3781f6bb2bdd143a56f408e442524926eR195-R201) since it also does essentially the same thing.